packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -800,9 +800,9 @@ export const components = (
       styleOverrides: {
         root: {
           abbr: {
-            borderBottomWidth: "1px",
-            borderBottomStyle: "dashed",
-            borderBottomColor: odysseyTokens.PalettePrimaryDark,
+            borderBottomWidth: odysseyTokens.BorderWidthMain,
+            borderBottomStyle: "dashed", // Token?
```

Comment:
@KevinGhadyani-Okta I'll be removing these comments when @taylorridgway-okta and I do the Token Review this week.